test/rtl/basic.mlir

```diff
@@ -3,43 +3,43 @@
 // CHECK-LABEL: func @test1(%arg0: i3) -> i50 {
 func @test1(%arg0: i3) -> i50 {
   // CHECK-NEXT:    %c42_i12 = rtl.constant(42 : i12) : i12
-  // CHECK-NEXT:    %[[RES0:.*]] = rtl.add %c42_i12, %c42_i12 : i12
-  // CHECK-NEXT:    %[[RES1:.*]] = rtl.mul %c42_i12, %[[RES0:.*]] : i12
+  // CHECK-NEXT:    [[RES0:%[0-9]+]] = rtl.add %c42_i12, %c42_i12 : i12
+  // CHECK-NEXT:    [[RES1:%[0-9]+]] = rtl.mul %c42_i12, [[RES0]] : i12
   %a = rtl.constant(42 : i12) : i12
   %b = rtl.add %a, %a : i12
   %c = rtl.mul %a, %b : i12
 
-  // CHECK-NEXT:    %[[RES2:.*]] = rtl.sext %arg0 : (i3) -> i7
-  // CHECK-NEXT:    %[[RES3:.*]] = rtl.zext %arg0 : (i3) -> i7
+  // CHECK-NEXT:    [[RES2:%[0-9]+]] = rtl.sext %arg0 : (i3) -> i7
+  // CHECK-NEXT:    [[RES3:%[0-9]+]] = rtl.zext %arg0 : (i3) -> i7
   %d = rtl.sext %arg0 : (i3) -> i7
   %e = rtl.zext %arg0 : (i3) -> i7
 
-  // CHECK-NEXT:    %[[RES4:.*]] = rtl.concat %[[C42_i12:.*]] : (i12) -> i12
+  // CHECK-NEXT:    [[RES4:%[0-9]+]] = rtl.concat %[[C42_i12:.*]] : (i12) -> i12
```

Comment:
I think this should be either `%c42_i12` or `%[[C42_i12]]` when also capturing the result of the constant operation above

Reply:
Yep, this should be `%c42_i12` instead of `%[[C42_i12:.*]]`

Reply:
Thanks! Missed that one